client/src/App.js

```diff
@@ -13,7 +14,9 @@ const App = () => (
 				<Route path="/about/this/site" element={<About />} />
 			</Routes>
 		</Layout>
+		<FindSchool />
```

Comment:
It will be a form to submit new schools, can we name the component "CreateSchool"?

Reply:
Done